modules/rostests/apitests/shell32/ItemIDList.cpp

```diff
@@ -202,3 +202,30 @@ START_TEST(PIDL)
         skip("?\n");
     ILFree(pidl);
 }
+
+START_TEST(ILIsEqual)
+{
+    LPITEMIDLIST p1, p2;
+
+    p1 = p2 = NULL;
+    ok_int(ILIsEqual(p1, p2), TRUE);
+
+    ITEMIDLIST emptyitem = {};
+    p1 = p2 = &emptyitem;
+    ok_int(ILIsEqual(p1, p2), TRUE);
+
+    p1 = SHCloneSpecialIDList(NULL, CSIDL_DRIVES, FALSE);
+    p2 = SHCloneSpecialIDList(NULL, CSIDL_DRIVES, FALSE);
+    if (p1 && p2)
+    {
+        ok_int(ILIsEqual(p1, p2), TRUE);
+        p1->mkid.abID[0] = 0x2E; // Convert Desktop RegItem to Computer RegItem
+        ok_int(ILIsEqual(p1, p2), FALSE);
+    }
+    else
+    {
+        skip("?\n");
```

Comment:
```suggestion
        skip("Failed to clone the IDs\n");
```
Actually, wouldn't it be of interest to test (with ok()) the p1 and p2 ?

Reply:
This would only fail if there is not enough memory to allocate about 20 bytes * 2 and not really part of the test but there is no way to declare what is just setup and what is the actual test AFAIK.

Reply:
Please change the skip message with something readable. (And the other one below as well.)